packages/notification/src/features/email/service..ts

```diff
@@ -112,29 +127,30 @@ async function markQueueRecordSuccess(record: NotificationQueueRecord) {
   return NotificationQueue.updateOne(
     { _id: record._id },
     {
-      status: 'success'
+      status: 'success',
+      $unset: {
+        error: ''
+      }
     }
   )
 }
 
-export async function loopNotificationQueue(request: Hapi.Request) {
-  const token = request.headers.authorization
-  const payload = request.payload as AllUsersEmailPayloadSchema
+export async function loopNotificationQueue(server: Hapi.Server) {
   isLoopInprogress = true
-  let record = await findOldestNotificationQueueRecord(payload.requestId)
+  let record = await findOldestNotificationQueueRecord()
   while (record) {
     logger.info(
       `Notification service: Initiating dispatch of notification emails for ${record.bcc.length} users`
     )
-    const res = await dispatch(record, token)
+    const res = await dispatch(record)
     if (!res.ok) {
       const error = await res.json()
       await markQueueRecordFailedWithErrorDetails(record, error)
-      request?.log(['error', error.error, internal(error.message)])
+      server.log(['error', error.error, internal(error.message)])
```

Comment:
Could this be a `logger.error` call so you don't have to pass in the whole Hapi server as a parameter?

Reply:
@rikukissa I intended to implement this https://www.npmjs.com/package/hapi-sentry#capturing-serverlog-and-requestlog-events to report in Sentry

Reply:
Ok, makes sense. What do you think about making the logger so that it would forward all `logger.error` calls to Sentry? Then we wouldn't have to integrate sentry so deeply to the HTTP server itself and we could receive other errors as well

Reply:
Could even be something like 

```ts
import pino from 'pino'

const logger = pino()

const originalError = logger.error.bind(logger)

logger.error = (err: string | Record<string, any>, msg: string) => {
  // Send to Sentry
  originalError(err, msg)
}

const level = process.env.NODE_ENV === 'test' ? 'silent' : process.env.LOG_LEVEL
if (level) {
  logger.level = level
}

```